clldutils/csvw/metadata.py

```diff
@@ -620,7 +620,7 @@ def check_referential_integrity(self, data=None, log=None):
                     else:
                         colrefs = [colref]
                     for colref in colrefs:
-                        if colref not in keys:
+                        if not (all(c is None for c in colref) or colref in keys):
```

Comment:
nit: how about ``if any(c is not None for c in colref) and colref not in keys``

Reply:
ok

Reply:
your version is 3 characters longer, though.